From 3f2afb29140c1caa97901e5e1cd38865afb19d6e Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Fri, 20 Jul 2012 16:24:55 +0200
Subject: [PATCH 2/2] tmpfiles: also look in /lib/tmpfiles.d on split /usr
 systems

https://bugs.freedesktop.org/show_bug.cgi?id=38686

I don't think the usecase case in that bug makes much sense, but all the
other tools do honour /lib in the search path so we probably should do
that here, too.
---
 src/tmpfiles/tmpfiles.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/tmpfiles/tmpfiles.c b/src/tmpfiles/tmpfiles.c
index 7a453dc..45125b7 100644
--- a/src/tmpfiles/tmpfiles.c
+++ b/src/tmpfiles/tmpfiles.c
@@ -107,6 +107,9 @@ static const char * const conf_file_dirs[] = {
         "/run/tmpfiles.d",
         "/usr/local/lib/tmpfiles.d",
         "/usr/lib/tmpfiles.d",
+#ifdef HAVE_SPLIT_USR
+        "/lib/tmpfiles.d",
+#endif
         NULL
 };
 
-- 
1.7.11.2

